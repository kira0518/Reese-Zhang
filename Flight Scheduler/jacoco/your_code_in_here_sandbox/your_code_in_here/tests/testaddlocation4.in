location
location add Dubbo -32.2811022 148.3241332 0.4
location Dubbo
location Sydney
exit

