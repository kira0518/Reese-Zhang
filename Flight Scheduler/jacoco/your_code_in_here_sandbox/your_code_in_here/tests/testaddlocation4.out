User: Usage:
LOCATION <name>
LOCATION ADD <name> <latitude> <longitude> <demand_coefficient>
LOCATION IMPORT/EXPORT <filename>

User: Successfully added location Dubbo.

User: Location:    Dubbo
Latitude:    -32.281102
Longitude:   148.324133
Demand:      +0.4000

User: Invalid location name.

User: Application closed.
